src/Symfony/Component/Security/Http/Logout/LogoutUrlGenerator.php

```diff
@@ -151,6 +151,10 @@ private function getListener(?string $key): array
             }
         }
 
+        if (null === $this->currentFirewallName) {
+            throw new \InvalidArgumentException('Unable to find the current firewall LogoutListener, please make sure request is behind a firewall.');
```

Comment:
"This request is not behind a firewall, please pass the firewall name manually to generate a logout URL."